webpack/webpack.config.babel.js

```diff
@@ -163,7 +163,8 @@ module.exports = {
       PluginTestUtils: absPath("src/js/plugin-bridge/PluginTestUtils"),
       "#EXTERNAL_PLUGINS": externalPluginsDir,
       "#PLUGINS": absPath("plugins"),
-      "#SRC": absPath("src")
+      "#SRC": absPath("src"),
+      "#TESTS": absPath("tests")
```

Comment:
@juliangieseke Where will this be used? Just curious, you can merge.

Reply:
just check where this PR is referenced :)